package by.academy.lesson12.exceptions.trycatch;

public class ExceptionDemo5 {
	public static void main(String[] args) {
//		try {
		// блок кода, в котором отслеживаются ошибки
//		} catch (ExceptionType1 e) {
		// обработчик исключений типа ExceptionType1
//		} catch (ExceptionType2 e) {
		// обработчик исключений типа ExceptionType2
//		} finally {
		// блок кода, который должен быть выполнен после завершения блока try
//		}
	}
}
